### DISCUSSION TITLE
PAGImageView加载非pag图片问题

### DISCUSSION THREAD

**Discussion by li-qingnan** [Tencent/libpag#1743] [Q&A] [answered]
看了PAGImageView是继承与UIImageView，所以就创建了PAGImageView用于加载普通图片和pag，
但是加载普通图片的时候，加载是没有问题的，但是app切到后台再切换回前台，图片显示就空白了。
加载pag图片的时候，app切到后台再切换回前台，图片显示就没有问题。

请问以上这是什么问题，该如何解决？

**Comment by domchen** [comment, marked as answer]
虽然可能可以这么用，但不要用PAGImageView去加载普通图片。本质上PAGImageView是个动图，有帧刷新的，从后台到前台的时候也会主动触发一次刷新，你没有设置PAGFile就会把内容清空。
